Improve plan documentation (#2755)
EnMasseProject/enmasse: documentation/assemblies/assembly-configuring.adoc

[id='configuring-{context}']
= Configuring {ProductName}

The service operator configures {ProductName} by defining resources constituting the "service configuration". This configuration contains instances of the following resource types:

* `AuthenticationService`  - Describes an authentication service instance used to authenticate messaging clients.

> * `AddressSpacePlan` - Describes the messaging resources available for address spaces using this plan, such as the available address plans, the amount of router and broker resources that can be used.

Reviewer: ```suggestion
* `AddressSpacePlan` - Describes the messaging resources available for address spaces using this plan, such as the available address plans and the amount of router and broker resources that can be used.
```

* `AddressPlan` - Describes the messaging resources consumed by a particular address using this plan, such as what fraction of routers and brokers an address will use and other properties that should be set for multiple addresses.

> * `StandardInfraConfig` - Describes the router and broker configuration for the `standard` address space type such as memory limits, storage capacity, affinity and more.

Reviewer: ```suggestion
* `StandardInfraConfig` - Describes the router and broker configuration for the `standard` address space type such as memory limits, storage capacity, affinity, and more.
```

> * `BrokeredInfraConfig` - Describes the broker configuration for the `brokered` address space type such as memory limits, storage capacity, affainity and more.

Reviewer: ```suggestion
* `BrokeredInfraConfig` - Describes the broker configuration for the `brokered` address space type such as memory limits, storage capacity, affinity, and more.
```

> When created, these resources defines the configuration that are available to the messaging tenants.

Reviewer: ```suggestion
When created, these resources define the configuration that is available to the messaging tenants.
```
